README.md

```diff
@@ -103,7 +103,7 @@ Some of the more important settings are as follows:
 
 ## Logging
 
-Logs are output to the `log/` dir.
+Logs are output to the `log/s` dir.
```

Comment:
`logs` instead of `log/s` ?

Reply:
yeah, that's odd

Reply:
I have dropped that commit from this PR.

Reply:
I have re-instated that commit, without my screwup :D